nuntium/templates/base_manager.html

```diff
@@ -13,7 +13,7 @@
     <meta name="description" content="">
     <meta name="author" content="">
     <link rel="shortcut icon" href="../../docs-assets/ico/favicon.png">
-    <link href='http://fonts.googleapis.com/css?family=Noto+Sans:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Noto+Sans:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
```

Comment:
I think _always_ serving from https is now the recommended approach. See [the update at the top of this article](http://www.paulirish.com/2010/the-protocol-relative-url/) for more details.